cli/src/lib/libDefs.js

```diff
@@ -64,11 +64,23 @@ async function rebaseCacheRepo(verbose?: VerboseOutput) {
 }
 
 /**
+ * Utility wrapper for ensureCacheRepo with an update expiry of 0 hours.
+ */
+async function updateCacheRepo(verbose?: VerboseOutput): Promise<boolean> {
+  try {
```

Comment:
Oh, but that detailed info doesn't actually contain the error message anyway... Yea, I still think we should just throw up to the toplevel.